README.md

```diff
@@ -1,50 +1,60 @@
+* Efficiency (emails should be fast to parse and triage)
+* Pretty (emails should look nice 😎)
+
+## Technical Information
+
 This daemon:
-1. Talks to Phabricator to get email event data
+1. [Talks to Phabricator to get email event data](https://github.com/mozilla-services/phabricator-extensions/tree/master/moz-extensions/src/email)
 2. Renders HTML and text emails based on this data
 3. Sends the rendered emails
 
-## Production Usage
+### Production Usage
 
-~This is a backend service that is deployed as a [dockerflow-compatible](https://github.com/mozilla-services/dockerflow)~
-~image called [`mozilla/phabricator-emails`](https://hub.docker.com/r/mozilla/phabext).~
-Until this image is image is deployed, you have to build it locally: `docker build .`. It should be 
-dockerflow-compatible already.
+This is a backend service that is deployed as a [dockerflow-compatible](https://github.com/mozilla-services/dockerflow)
+image called [`mozilla/phabricator-emails`](https://hub.docker.com/r/mozilla/phabricator-emails).
 
-### Configuration
+#### Configuration
 
 Place a `settings.ini` file in the `/app` directory of the container.
 There's an example production `settings.ini` file in the repository called `settings.production-example.ini`. You can
 copy that one, then consult the [configuration documentation below](#configuration-reference) for additional details.
 Finally, set an environment variable called `PHABRICATOR_EMAILS_SETTINGS_PATH` to `/app/settings.ini`.
 
-### Amazon credentials
+#### Amazon credentials
 
 To follow Amazon convention, the AWS credentials can be provided via 
 [environment variable](https://boto3.amazonaws.com/v1/documentation/api/latest/guide/configuration.html#environment-variables)
 or via `settings.ini`, whichever is more convenient.
 
-### First-time preparations
+#### First-time preparations
 
-The database has to be populated with tables and be synchronized with the current email status of Phabricator.
+The database has to be populated with tables and synchronized with the current feed status of Phabricator.
 Once `settings.ini` has been set up, you can perform this initialization with: `docker run $container prepare`
 
-### Run
+#### Run
 
 Once first-time preparation is finished, and you're ready to run, do `docker run $container service`.
-Unless you're using `[dev]` settings, it will loop every `[phabricator].poll_gap_seconds` seconds, query
+It will loop every `[phabricator].poll_gap_seconds` seconds, query
 Phabricator for email events, and send them using the driver you specified with an `[email-*]` section.
 
-### Updating deployments
+#### Updating deployments
 
 There may be changes to the the database schema, so you'll want to run `phabricator-emails migrate` to upgrade
 the database schema after updating the `phabricator-emails` code.
 
-## Development
+### Development
 
 For testing locally, you're going to need a couple things:
-1. Python 3
+1. Python 3.9.
```

Comment:
I think in the future it would be good to have a `docker-compose.yml` file with some local settings, so that developers can use that for local development/testing instead of setting things up in a virtual environment (and with very specific Python versions).

Reply:
Makes sense to me :+1: